apps/sources/api/types.py

```diff
@@ -0,0 +1,19 @@
+import graphene
+
+from apps.graph.types import AbstractModuleType
+from apps.sources.models import Source
+
+
+class SourceType(AbstractModuleType):
+    """GraphQL type for the Source model."""
+
+    slug = graphene.String()
+    citation_html = graphene.String()
+    citation_string = graphene.String()
+    attributee_html = graphene.String()
+    title = graphene.String()
+    model = graphene.String()
```

Comment:
Isn't this inherited? https://github.com/ModularHistory/modularhistory/blob/6ee5a052b250c0e24893fd635196d8acde90d026/apps/graph/types.py#L5-L8

Reply:
Ah, yes, that's a relic from immediately before I implemented AbstractModuleType.